foundation-ui/utils/__tests__/ReactUtil-test.js

```diff
@@ -0,0 +1,43 @@
+jest.dontMock('../ReactUtil');
+/* eslint-disable no-unused-vars */
+const React = require('react');
+/* eslint-enable no-unused-vars */
+const TestUtils = require('react-addons-test-utils');
+
+const ReactUtil = require('../ReactUtil');
+
+describe('ReactUtil', function () {
+
+  it('should wrap the elements if there is more than one', function () {
+    const result = ReactUtil.wrapElements(
+        [<span key="1">test</span>, <span key="2">test</span>]
+    );
+
+    expect(TestUtils.isElementOfType(result, 'div')).toEqual(true);
+  });
+
+  it('should not wrap if not necessary', function () {
+    const elements = ReactUtil.wrapElements(<span>test</span>);
+
+    expect(TestUtils.isElementOfType(elements, 'span')).toEqual(true);
```

Comment:
@wavesoft and I discussed this one offline and agreed that we don't need an extra test case, but might need to adjust the tag names to avoid confusions.